pyomo/contrib/mindtpy/util.py

```diff
@@ -723,125 +721,27 @@ def copy_var_list_values_from_solution_pool(
 
 
 class GurobiPersistent4MindtPy(GurobiPersistent):
-    """A new persistent interface to Gurobi.
+    """A new persistent interface to GUROBI.
```

Comment:
I think 'Gurobi' is more correct than 'GUROBI'--I think revert these changes.

Reply:
Totally right, let's keep CPLEX and Gurobi

Reply:
I thought you were going to change these back to 'Gurobi' rather than 'GUROBI'? (It's not an acronym, so I think it shouldn't be in all caps.)

Reply:
Yes. I changed `GUROBI` to `Gurobi`. Thanks for reminding me.